doc/classes/SkeletonProfile.xml

```diff
@@ -160,8 +161,12 @@
 	</methods>
 	<members>
 		<member name="bone_size" type="int" setter="set_bone_size" getter="get_bone_size" default="0">
+			The amount of [code]bone_map[/code] bones used in advanced import retargeting
+			[b]Note:[/b] For example, [SkeletonProfileHumanoid] has 56 bones.
 		</member>
 		<member name="group_size" type="int" setter="set_group_size" getter="get_group_size" default="0">
+			The amount of [code]bone_map[/code] groups of bones used in advanced import retargeting
+			[b]Note:[/b] For example, [SkeletonProfileHumanoid] has 4 groups.
```

Comment:
Missing period at end of line. The example can come right after the first sentence, it doesn't need to be a "Note:".

What is `bone_map`? It's never defined, and referring to something with code formatting only makes sense if users can map it to something they know which is written exactly like this in their code.

Reply:
I understand the confusion behind `bone_map`. Like I stated in the squashed commit message and PR, I am just trying to be consistent with the documentation of [Retargeting 3D Skeletons](https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/retargeting_3d_skeletons.html#bone-map).
Should the line:

> When you select the Skeleton3D node in the advanced scene import menu, a menu will appear on the right-hand side containing the "Retarget" section. The Retarget section has a single property `bone_map`.

be reworked in [Retargeting 3D Skeletons](https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/retargeting_3d_skeletons.html#bone-map)?

Reply:
You can rewrite it `[BoneMap]` as link for defined class.

Reply:
Ok! Just did and pushed to commit.